server/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <threadsafe>false</threadsafe>
+  <sessions-enabled>true</sessions-enabled>
+  <runtime>java8</runtime>
+  <static-files>
```

Comment:
What is the web preview server? Is it easy to disable this for the released app? Like you could have a dev web xml and prod web xml?

Reply:
For now my solution was to just `npm start` in `ui/`; because I knew I'd do what you described eventually but I was thinking that it'd take a slightly significant amount of time & people wanted this scaffold so they could begin working; I could do this now?